### PR TITLE
https://issues.redhat.com/browse/ACM-4806 Improving endpoint explanation for obs

### PR DESCRIPTION
**Pull request by dockerymick**
https://issues.redhat.com/browse/ACM-4806

### CODE REVIEW

File: observability/observability_enable.adoc [hunk 1]
```diff
@@ -93,15 +93,17 @@ stringData:
...
     config:
       bucket: YOUR_S3_BUCKET
-      endpoint: YOUR_S3_ENDPOINT
+      endpoint: YOUR_S3_ENDPOINT <1>
       insecure: true
       access_key: YOUR_ACCESS_KEY
       secret_key: YOUR_SECRET_KEY
 ----
 +
+<1> For HTTPS, enter the hostname without the fully qualified domain name (FQDN). Enter the URL for your Amazon S3 endpoint that might resemble the following URL: `example.redhat.com:443`.
```
Comment (dockerymick) [inline observability/observability_enable.adoc]:
```suggestion
<1> Enter the URL without the protocol. Enter the URL for your Amazon S3 endpoint that might resemble the following URL: `example.redhat.com:443`.
```

File: observability/observability_enable.adoc [hunk 2]
```diff
@@ -161,13 +163,17 @@ stringData:
...
       bucket: YOUR_RH_DATA_FOUNDATION_BUCKET
-      endpoint: YOUR_RH_DATA_FOUNDATION_ENDPOINT
+      endpoint: YOUR_RH_DATA_FOUNDATION_ENDPOINT <1>
       insecure: false
       access_key: YOUR_RH_DATA_FOUNDATION_ACCESS_KEY
       secret_key: YOUR_RH_DATA_FOUNDATION_SECRET_KEY
 ----
 +
-For more details, see link:https://www.redhat.com/en/technologies/cloud-computing/openshift-data-foundation[Red Hat OpenShift Data Foundation]. For Red Hat OpenShift on IBM (ROKS), your secret might resemble the following file:
+<1> For HTTPS, enter the hostname without the fully qualified domain name (FQDN). Enter the URL for your Red Hat OpenShift Data Foundation endpoint that might resemble the following URL: `example.redhat.com:443`.
```
Comment (dockerymick) [inline observability/observability_enable.adoc]:
```suggestion
<1> Enter the URL without the protocol. Enter the URL for your Red Hat OpenShift Data Foundation endpoint that might resemble the following URL: `example.redhat.com:443`.
```

File: observability/observability_enable.adoc [hunk 3]
```diff
@@ -182,21 +188,23 @@ stringData:
...
     config:
       bucket: YOUR_ROKS_S3_BUCKET
-      endpoint: YOUR_ROKS_S3_ENDPOINT
+      endpoint: YOUR_ROKS_S3_ENDPOINT <1>
       insecure: true
       access_key: YOUR_ROKS_ACCESS_KEY
       secret_key: YOUR_ROKS_SECRET_KEY
 ----
 +
+<1> For HTTPS, enter the hostname without the fully qualified domain name (FQDN). Enter the URL for your Red Hat OpenShift Data Foundation endpoint that might resemble the following URL: `example.redhat.com:443`.
```
Comment (dockerymick) [inline observability/observability_enable.adoc]:
```suggestion
<1> Enter the URL without the protocol. Enter the URL for your Red Hat OpenShift Data Foundation endpoint that might resemble the following URL: `example.redhat.com:443`.
```

File: observability/observability_enable.adoc [hunk 4]
```diff
@@ -328,7 +336,7 @@ stringData:
    endpoint: s3.$REGION.amazonaws.com
    signature_version2: false
 ----
-
+//need to verify that the endpoint value is valid^
```
Comment (dockerymick) [inline observability/observability_enable.adoc]:
```suggestion

```

File: release_notes/known_issues_observability.adoc [hunk 5]
```diff
@@ -130,3 +130,21 @@ addon.open-cluster-management.io/search_memory_limit=1024Mi
 //2.8:ACM-4942
 
 The Grafana dashboard shows an empty label list if the `disableHubSelfManagement` parameter is set to `true` in the `mulitclusterengine` custom resource. You must set the parameter to `false` or remove the parameter to see the label list. See link:../install/adv_config_install.adoc#disable-hub-self-management[disableHubSelfManagement] for more details.
+
+[fqdn-not-supported]
+=== Endpoint URL cannot have fully qualified domain names (FQDN)
+//2.7:ACM-4806
+
+When you use the FQDN and or protocol for the `endpoint` parameter, your observability pods are not enabled. The following error message is displayed:
```
Comment (dockerymick) [inline release_notes/known_issues_observability.adoc]:
```suggestion
When you use the FQDN or protocol for the `endpoint` parameter, your observability pods are not enabled. The following error message is displayed:
```

File: release_notes/known_issues_observability.adoc [hunk 6]
```diff
@@ -130,3 +130,21 @@ addon.open-cluster-management.io/search_memory_limit=1024Mi
...
+
+[source,bash]
+----
+Endpoint url cannot have fully qualified paths
+----
+
+Your `endpoint` value must resemble the following URL for your secrets:
```
Comment (dockerymick) [inline release_notes/known_issues_observability.adoc]:
```suggestion
Enter the URL without the protocol. Your `endpoint` value must resemble the following URL for your secrets:
```

File: observability/observability_enable.adoc [hunk 7]
```diff
@@ -182,21 +188,23 @@ stringData:
...
++
 For more details, follow the IBM Cloud documentation, link:https://cloud.ibm.com/objectstorage/create[Cloud Object Storage]. Be sure to use the service credentials to connect with the object storage. For more details, follow the IBM Cloud documentation, link:https://cloud.ibm.com/objectstorage/create%5BCloud[Cloud Object Store] and link:https://cloud.ibm.com/docs/cloud-object-storage/iam?topic=cloud-object-storage-service-credentials%5BService[Service Credentials].
 
-** For Amazon S3 or S3 compatible storage, you can also use short term, limited-privilege credentials generated with AWS Security Token Service (AWS STS). Refer to link:https://docs.aws.amazon.com/STS/latest/APIReference/welcome.html[AWS Security Token Service documentation] for more details.
+- For Amazon S3 or S3 compatible storage, you can also use short term, limited-privilege credentials generated with AWS Security Token Service (AWS STS). Refer to link:https://docs.aws.amazon.com/STS/latest/APIReference/welcome.html[AWS Security Token Service documentation] for more details.
```
Comment (swopebe) [inline observability/observability_enable.adoc]: that are generated?

**Review by swopebe — APPROVED** [review]
one small comment.

File: observability/observability_enable.adoc (hunk 7 above)
Comment (dockerymick) [inline observability/observability_enable.adoc]: Thanks ! 
```suggestion
- For Amazon S3 or S3 compatible storage, you can also use short term, limited-privilege credentials that are generated with AWS Security Token Service (AWS STS). Refer to link:https://docs.aws.amazon.com/STS/latest/APIReference/welcome.html[AWS Security Token Service documentation] for more details.
```

File: observability/observability_enable.adoc [hunk 8]
```diff
@@ -120,8 +122,8 @@ stringData:
 ----
 +
 For more details, see link:https://cloud.google.com/storage/docs/introduction[Google Cloud Storage].
-
-** For Azure your secret might resemble the following file:
+//is this correct? Is the endpoint value valid?
```
Comment (subbarao-meduri) [inline observability/observability_enable.adoc]: Based on [Thanos documentation](https://thanos.io/tip/thanos/storage.md/#azure) 

If msi_resource is used, authentication is done via system-assigned managed identity. The value for Azure should be https://\<storage-account-name\>.blob.core.windows.net.

If user_assigned_id is used, authentication is done via user-assigned managed identity. When using user_assigned_id the msi_resource defaults to https://\<storage_account\>.\<endpoint\>

**Review by subbarao-meduri — APPROVED** [review]
/lgtm

File: observability/observability_enable.adoc [hunk 9]
```diff
@@ -138,15 +140,18 @@ stringData:
...
       storage_account_key: YOUR_STORAGE_KEY
       container: YOUR_CONTAINER
-      endpoint: blob.core.windows.net
+      endpoint: blob.core.windows.net <1>
       max_retries: 0
 ----
 +
+<1> If you use the `msi_resource` path, the endpoint authentication is complete by using the system-assigned managaed identity. Your value must resemble the following endpoint: `https://<storage-account-name>.blob.core.windows.net`. 
++
+If you use the `user_assigned_id` pat, endpoint authentication is complete by using the user-assigned managed identity. When you use the `user_assigned_id`, the `msi_resource` endpoint default value is `https:<storage_account>.<endpoint>`.
```
Comment (dockerymick) [inline observability/observability_enable.adoc]:
```suggestion
If you use the `user_assigned_id` path, endpoint authentication is complete by using the user-assigned managed identity. When you use the `user_assigned_id`, the `msi_resource` endpoint default value is `https:<storage_account>.<endpoint>`.
```

**Comment by dockerymick** [comment]
Received review and approval from peer and dev. Thanks! Merging this PR

**Pull request merged**